### PR TITLE
PW-382 Set IssuerId in checkout

### PR DESCRIPTION
**Pull request by cyattilakiss**
**Description**
Remove IDEAL check when filtering the available payment methods
Let other payment methods also use the issuer drop down list

### CODE REVIEW

File: view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js [hunk 1]
```diff
@@ -152,7 +152,7 @@ define(
                         fullScreenLoader.stopLoader();
                     }
                 ).fail(function (error) {
-                    console.log(JSON.stringify(error));
+                    //console.log(JSON.stringify(error));
```
Comment (rikterbeek) [inline view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js]: let's remove this line at all.

File: view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js [hunk 2]
```diff
@@ -193,10 +193,22 @@ define(
...
+
+                    if (value.hasOwnProperty("issuers")) {
+                        if (value.issuers.length == 0) {
+                            return false;
+                        }
+
                         result.issuerIds = value.issuers;
                         result.issuerId = ko.observable(null);
-                    } else if (value.isPaymentMethodOpenInvoiceMethod) {
+                    }else if (value.isPaymentMethodOpenInvoiceMethod) {
```
Comment (rikterbeek) [inline view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js]: what formatter do you use? Think we can keep this space.

Comment (cyattilakiss) [reply view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js]: Nothing extraordinary I usually follow how they style the js code:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else
If we have a common setting I can set it up 👍

File: view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js (hunk 1 above)
Comment (cyattilakiss) [reply view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js]: agree

**Pull request merged**